### DISCUSSION TITLE
Where is the dbt Cloud CLI installation path when installing it via the PowerUser setup wizard?

### DISCUSSION THREAD

**Discussion by jairus-m** [AltimateAI/vscode-dbt-power-user#1141] [Q&A]
Before PowerUser's support for the Cloud CLI, I had a virtual environment that had dbt-core and I would point PowerUser to that while also having the physical dbt Cloud CLI binaries installed locally. I made an alias that pointed to the Cloud CLI binary so that i can run "dbt-cloud build" without having conflict with dbt-core.

Now that there's support for dbt Cloud CLI + PowerUser, I went ahead and activated that setting in the extension. It did not detect the dbt-cloud binary that I already had installed. However, the setup wizard did give me an option to install dbt Cloud CLI into my current environment.

This may be a silly question, but where is the new installation located on my machine/environment?

**Comment by mdesmet** [comment]
The extension uses pip to install it into the selected Python Interpreter. As both dbt-core and dbt (cloud) install an executable `dbt` into the Python interpreter bin directory, it is not supported to have both dbt-core and cloud installed within the same Python interpreter, only one of them will work.

**Reply by jairus-m** [reply]
Thanks for the reply. So if I had my virtual environment w/ the attributed Python interpreter activated and then had PowerUser install dbt Cloud CLI, the ```dbt.exe``` file would be installed within that virutal environment?

For my case, it looks like I found it:

```"C:\Users\JairusMartinez\.virtualenvs\dbt_dev-PLb3_7rL\Scripts\dbt.exe"``` 

Does this look correct to you?

Before I resolve this, I do have one more question. For keeping dbt Cloud CLI updated to the latest version, does PowerUser take care of that?

Thanks.
